### DISCUSSION TITLE
Recover the vocals separately? With a plugin?

### DISCUSSION THREAD

**Discussion by Valmara69** [Anjok07/ultimatevocalremovergui#1079] [Q&A]
Hello to all the community! To the passenger, happy new year and good health to all.

So my question is this:
Is it possible to recover the vocals in another separate file via one of the plugins or are the vocals irretrievably lost in the process?

_I add that I have not yet used the program but I installed it without much difficulty under Debian Linux bullseye+backports_

**Comment by MrRorls12** [comment]
Just download a model and select "Vocals only"
